polkadot-parachains/canvas-kusama/src/xcm_config.rs

```diff
@@ -170,7 +176,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmExecuteFilter = Nothing;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type XcmTeleportFilter = Everything;
-	type XcmReserveTransferFilter = Everything;
+	type XcmReserveTransferFilter = EverythingBut<IsParent>;
```

Comment:
I would add some docs here explaining why, like:
```suggestion
	/// Statemine acts as a `NativeAsset` reserve to all chains except the Relay Chain, which
	/// recognizes no reserves.
	type XcmReserveTransferFilter = EverythingBut<IsParent>;
```